packages/newspringchurchapp/src/ui/TileImage/index.js

```diff
@@ -11,7 +11,7 @@ const CardView = styled(
     overflow: 'hidden',
     width: '100%',
     height: '100%',
-    aspectRatio: 1,
+    aspectRatio: 1.75,
```

Comment:
it's strange how this isn't used anywhere else. should this component be included in the prayer folder if not?

Reply:
I guess a better question is are we sure this isn't used anywhere else?

Reply:
I just looked and it's not. But ... there is one use from core and one that is in another file that actually defines _another_ `TileImage`. This is ripe for a refactor.